pkg/secrets.go

```diff
@@ -0,0 +1,59 @@
+package pkg
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	apierrors "k8s.io/kubernetes/pkg/api/errors"
+	kcl "k8s.io/kubernetes/pkg/client/unversioned"
+	"os"
+)
+
+var secretName = "private-registry" + "-" + os.Getenv("DEIS_REGISTRY_LOCATION")
```

Comment:
I think we should turn this secret name prefix into an env var so we can treat it as configuration rather than having to recompile. I think this also goes for controller code as well!